tensorboard/components/tf_tensorboard/tf-tensorboard.html

```diff
@@ -222,9 +202,14 @@
         // Which tab is selected (scalars, graph, images etc).
         mode: {
           type: String,
-          computed: '_getModeFromIndex(modeIndex)',
+          computed: '_getModeFromIndex(tabs, modeIndex)',
```

Comment:
Sure: that makes sense, as "plugin name" is the data and "tab" is {the,one particular} representation.